### DISCUSSION TITLE
PSAVanCanBridgeHW

### DISCUSSION THREAD

**Discussion by igorguv** [morcibacsi/PSAVanCanBridge#39] [Q&A]
Добрый вечер, как вы думаете, будет ли работать ваш проект на Peugeot 607 с шиной VAN 2001 года выпуска? и как можно подключить van-can мост к андроид магнитоле  TS7?

**Comment by morcibacsi** [comment]
If you want to connect to an Android unit I suggest to buy an adapter like this. In this case you don't need my hardware. This one supports VAN bus: https://www.aliexpress.com/item/1005004930341941.html

**Comment by AndrewMal1982** [comment]
Good day. Might you can help me to find this adapter to purchase? 
At the moment I have peugeot 607 2003. Two years ago I install RT4 via VanCan Bridge(full kit for installation was purchased). But the bridge work not correct, I suppose. RT4 not started few times. After 2-10 cycles of ignition on/off, the radio get started.
